token/js/src/actions/transfer.ts

```diff
@@ -28,13 +31,27 @@ export async function transfer(
     amount: number | bigint,
     multiSigners: Signer[] = [],
     confirmOptions?: ConfirmOptions,
-    programId = TOKEN_PROGRAM_ID
+    programId = TOKEN_PROGRAM_ID,
+    memoProgramId = MEMO_PROGRAM_ID,
+    memo?: string
```

Comment:
Instead of adding these as more complicated / undefined variables, let's make this clearer through a new `actions` file called `transferWithMemo`.  There, you can make the memo mandatory